src/main/java/it/gov/pagopa/pu/registry/repository/DebtPositionRegistryRepository.java

```diff
@@ -0,0 +1,9 @@
+package it.gov.pagopa.pu.registry.repository;
+
+import it.gov.pagopa.pu.registry.model.DebtPositionRegistry;
+import org.springframework.data.mongodb.repository.MongoRepository;
+import org.springframework.data.rest.core.annotation.RepositoryRestResource;
+
+@RepositoryRestResource(collectionResourceRel = "debtPositionRegistries", path = "debt-position-registry")
```

Comment:
the path should be in the plural form 
```suggestion
@RepositoryRestResource(collectionResourceRel = "debtPositionRegistries", path = "debt-position-registries")
```

Reply:
I've picked "registry" because in this case it's a "registry" of installment events and not multiple registries of installment events - made the change

Reply:
we use the plural form for the part in order to be more Restfull
eg:
GET /.../...registries to fetch all